app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -68,6 +68,7 @@ CONFIG_DEBUG_COREDUMP_MEMORY_DUMP_MIN=y
 CONFIG_HEAP_MEM_POOL_SIZE=8192
 CONFIG_LLEXT=y
 CONFIG_LLEXT_STORAGE_WRITABLE=y
+CONFIG_LLEXT_EXPERIMENTAL=y
```

Comment:
why call it experimental? Perhaps LLEXT_SAVE_RESTORE makes better sense?

Reply:
@ranj063 we don't, Zephyr does. This option has recently been added to Zephyr to support our partial LLEXT restore, but the way how we added it there isn't perfectly clean. It's hardly usable by other LLEXT users, it's almost SOF-specific, therefore it's been decided to mark it "experimental" until a proper universal solution replaces it.